File: service/src/main/java/com/pauldaniv/template/service/TestOneServiceImpl.java
package com.pauldaniv.template.service;

import com.pauldaniv.template.api.TestOneService;
import com.pauldaniv.template.lib.first.util.TestUtil;
import com.pauldaniv.template.request.TestOne;
import org.springframework.stereotype.Service;

@Service
public class TestOneServiceImpl implements TestOneService {
    @Override
    public TestOne getIt() {
        return new TestOne(TestUtil.join("first", "second"));
    }
}

File: service/src/main/kotlin/com/pauldaniv/template/service/TestOneServiceImpl.kt
package com.pauldaniv.template.service

import com.pauldaniv.template.api.TestOneService
import com.pauldaniv.template.lib.first.util.TestUtil
import com.pauldaniv.template.request.TestOne
import org.springframework.stereotype.Service

@Service
class TestOneServiceImpl : TestOneService {
  override fun getIt(): TestOne {
    return TestOne(TestUtil.join("first", "second"))
  }
}
